common/workunit/workunit.cpp

```diff
@@ -5765,7 +5707,7 @@ void CLocalWorkUnit::setStatistic(StatisticCreatorType creatorType, const char *
         mergeAction = StatsMergeAppend;
     }
 
-    if (mergeAction != StatsMergeAppend)
+    if (mergeAction != StatsMergeAppend) // RKC->GH Is this right??
```

Comment:
I think it is correct, but part of the stats that doesn't feel quite right.  If appending the new values aren't combined with the existing ones (see code above which means that a new node is always created).

I think timestamps should possibly be revisited....